build.gradle

```diff
@@ -119,30 +119,26 @@ repositories {
   maven { url 'https://jitpack.io' }
 }
 
-def versions = [
-  spring : '3.2.4'
-]
-
-configurations.all {
-  resolutionStrategy.eachDependency { DependencyResolveDetails details ->
-    if (details.requested.group == 'org.apache.logging.log4j') {
-      details.useVersion '2.15.0'
-    }
-  }
+ext {
+  log4JVersion = '2.23.1'
+  spring = '3.2.5'
 }
 
 dependencies {
-  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-json', version: "${versions.spring}"
-  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa', version: "${versions.spring}"
-  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-oauth2-client', version: "${versions.spring}"
+  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-json', version: spring
```

Comment:
I wonder why we are not using the springboot plugin to manage springboot dependency like the other backend projects. With that, we do not need to add the version string for the springboot dependency everywhere